pkg/router/template/router.go

```diff
@@ -153,7 +153,11 @@ func newTemplateRouter(cfg templateRouterCfg) (*templateRouter, error) {
 		return "templaterouter", nil
 	}
 
-	numSeconds := int(cfg.reloadInterval.Seconds())
+	numSeconds := int(router.reloadInterval.Seconds())
+	if numSeconds < 0 {
+		numSeconds = 5
+		glog.Errorf("Router --interval set < 0. Using 5 second reload interval")
```

Comment:
If I've rejected an explicit setting, I think I'd prefer to fail them as opposed to simply choosing a different value.

Is this for backwards compatibility?  If so, when would we tighten it?